docs/content/guides/tablespaces.md

```diff
@@ -0,0 +1,307 @@
+---
+title: "Tablespaces in PGO"
+date:
+draft: false
+weight: 160
+---
+
+{{% notice warning %}}
+PGO tablespaces is currently in `Alpha` and may interfere with other features.
+(See below for more details.)
+{{% /notice %}}
+
+A [Tablespace](https://www.postgresql.org/docs/current/manage-ag-tablespaces.html)
+is a Postgres feature that is used to store data on a different volume than the
+primary data directory. While most workloads do not require tablespaces, they can
+be helpful for larger data sets or utilizing particular hardware to optimize
+performance on a particular Postgres object (a table, index, etc.). Some examples
+of use cases for tablespaces include:
+
+- Partitioning larger data sets across different volumes
+- Putting data onto archival systems
+- Utilizing faster/more performant hardware (or a storage class) for a particular database
+- Storing sensitive data on a volume that supports transparent data-encryption (TDE)
+
+and others.
+
+In order to use Postgres tablespaces properly in a highly-available,
+distributed system, there are several considerations to ensure proper operations:
+
+- Each tablespace must have its own volume; this means that every tablespace for
+every replica in a system must have its own volume;
+- The filesystem map must be consistent across the cluster;
```

Comment:
:question: How does one check this?

Reply:
Reading that now, it seems unclear and I dislike `cluster` there; what I mean here is that every postgres pod that is part of the postgres cluster needs to have the same paths, e.g., if you start a postgres (leader) and create a tablespace at `/tablespaces/user/data` that command is going to be replayed on any replicas you have, so that path has to be valid for every pod.